GitUpKit/Core/GCFunctions.m

```diff
@@ -95,7 +95,7 @@ BOOL GCGitOIDFromSHA1Prefix(NSString* prefix, git_oid* oid, NSError** error) {
 }
 
 NSString* GCUserFromSignature(const git_signature* signature) {
-  return [NSString stringWithFormat:@"%@ <%s>", [NSString stringWithUTF8String:signature->name], signature->email];
+  return [NSString stringWithFormat:@"%@ <%s>", [NSString stringWithUTF8String:signature->name], signature -> email];
```

Comment:
I'm confused by this one. We have 2 different styles on this same line. I'd prefer the previous style.

Reply:
Confused as well. Will look.